schemas/2018-08-01/Microsoft.Network.json

```diff
@@ -4,6 +4,163 @@
   "title": "Microsoft.Network",
   "description": "Microsoft Network Resource Types",
   "resourceDefinitions": {
+    "frontDoors": {
+      "name": "string",
+      "type": "Microsoft.Network/frontDoors",
+      "apiVersion": "2018-08-01",
+      "location": "string",
+      "tags": {},
+      "properties": {
+        "friendlyName": "string",
+        "routingRules": [
```

Comment:
>routingRules [](start = 9, length = 12)

are these manually written?

Reply:
I used the autorest tool to generate the schemas.